src/Adapter/MSTest.TestAdapter/Helpers/INotCachedAttributeHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Reflection;
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Helpers;
+
+internal interface INotCachedAttributeHelper
```

Comment:
Would deserve some comment about what this is needed for.

Also, I would avoid `Helper` suffix for an interface

Reply:
INotCachedReflectionAccessor is better?